dm/dm/master/server_test.go

```diff
@@ -2031,16 +2032,29 @@ func (t *testMaster) TestGetCfg(c *check.C) {
 	c.Assert(resp1.Result, check.IsTrue)
 	c.Assert(strings.Contains(resp1.Cfg, "name: test"), check.IsTrue)
 
-	// wrong task name
+	// not exist task name
+	notExistTaskName := "wrong"
 	req2 := &pb.GetCfgRequest{
-		Name: "haha",
+		Name: notExistTaskName,
 		Type: pb.CfgType_TaskType,
 	}
 	resp2, err := server.GetCfg(context.Background(), req2)
 	c.Assert(err, check.IsNil)
 	c.Assert(resp2.Result, check.IsFalse)
 	c.Assert(resp2.Msg, check.Equals, "task not found")
 
+	// test get task template
+	openapiTask, err := fixtures.GenNoShardOpenAPITaskForTest()
+	openapiTask.Name = notExistTaskName
+	c.Assert(err, check.IsNil)
+	c.Assert(ha.PutOpenAPITaskTemplate(t.etcdTestCli, openapiTask, true), check.IsNil)
+	c.Assert(failpoint.Enable("github.com/pingcap/tiflow/dm/dm/master/MockSkipAdjustTargetDB", `return(true)`), check.IsNil)
+	resp2, err = server.GetCfg(context.Background(), &pb.GetCfgRequest{Name: notExistTaskName, Type: pb.CfgType_TaskTemplateType})
+	c.Assert(failpoint.Disable("github.com/pingcap/tiflow/dm/dm/master/MockSkipAdjustTargetDB"), check.IsNil)
+	c.Assert(err, check.IsNil)
+	c.Assert(resp2.Result, check.IsTrue)
+	c.Assert(strings.Contains(resp2.Cfg, fmt.Sprintf("name: %s", notExistTaskName)), check.IsTrue)
```

Comment:
why it's named `notExistTaskName`, now we are expected it existing.

Reply:
don't forget this comment

Reply:
addressed in https://github.com/pingcap/tiflow/pull/3991/commits/1be01edcb9609c10442f22146281d4c098ab83d6 and https://github.com/pingcap/tiflow/pull/3991/commits/a09177f635b30a2d9348c309083ef35a31a86452